internal/generator/vector/output/common/vector_secret.go

```diff
@@ -0,0 +1,36 @@
+package common
+
+import (
+	"fmt"
+	"github.com/openshift/cluster-logging-operator/internal/generator/framework"
+)
+
+type VectorSecret struct {
```

Comment:
Consider dropping "Vector" from this name since its already under a "vector" package